### PR TITLE
Add scrollbar support; fix navigation when multiples slider are used …

### PR DESCRIPTION
**Pull request by leonardofaria**
I added support to scrollbar option and I also fixed the navigation (prev/next). These buttons were propagated to all swipers of the page.

### CODE REVIEW

**Comment by leonardofaria** [comment]
I'm trying to solve the tests issue, but even without my commit the `ember test` breaks.

I'm not sure if I'm missing something about this.

File: addon/components/swiper-container.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ export default Ember.Component.extend({
     }
 
     if (this.get('navigation')) {
-      options.nextButton = '.swiper-button-next';
-      options.prevButton = '.swiper-button-prev';
+      options.nextButton = `#${this.get('elementId')} .swiper-button-next`;
```
Comment (Suven) [inline addon/components/swiper-container.js]: could you change this to this.$('.swiper-button-next') instead?

File: addon/components/swiper-container.js [hunk 2]
```diff
@@ -59,11 +59,16 @@ export default Ember.Component.extend({
     if (this.get('grabCursor')) {
       options.grabCursor = true;
     }
-    
+
     if (this.get('breakpoints')) {
       options.breakpoints = this.get('breakpoints');
     }
 
+    if (this.get('scrollbar')) {
```
Comment (Suven) [inline addon/components/swiper-container.js]: I think this should go into a different PR. One PR = one purpose

File: addon/components/swiper-container.js [hunk 3]
```diff
@@ -59,11 +59,16 @@ export default Ember.Component.extend({
...
+      options.scrollbar = `#${this.get('elementId')} .swiper-scrollbar`;
```
Comment (Suven) [inline addon/components/swiper-container.js]: same as above: `this.$('.swiper-scrollbar')` instead
